backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc

```diff
@@ -14,9 +14,16 @@ You back up applications by creating a `Backup` custom resource (CR). See xref:.
 
 For more information about CSI volume snapshots, see xref:../../../storage/container_storage_interface/persistent-storage-csi-snapshots.adoc#persistent-storage-csi-snapshots[CSI volume snapshots].
 
-:FeatureName: The `CloudStorage` API for S3 storage
+:FeatureName: The `CloudStorage` API, which automates the creation of a bucket for object storage,
 include::snippets/technology-preview.adoc[]
 
+[NOTE]
+====
+The `CloudStorage` API supports manually creating a `BackupStorageLocation` object specifying an existing S3 bucket. The `CloudStorage` API that creates an S3 bucket automatically, currently is only enabled for AWS S3 storage.
+
+The `CloudStorage` API is a Technology Preview, if you use a `CloudStorageLocation` object and want OADP to use the `CloudStorage` API to automatically create an S3 bucket, for use as a `BackupStorageLocation` with GCP and Microsoft Azure storage.
```

Comment:
I think it should be CloudStorage instead of cloudStorageLocation

Reply:
@PrasadJoshi12 
Thank you for pointing that out. I updated it.

Reply:
Thanks @CarmiWisemon  !